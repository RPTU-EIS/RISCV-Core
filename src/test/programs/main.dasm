   0:	00004137          	lui	sp,0x4
   4:	ffc10113          	addi	sp,sp,-4 # 3ffc <_data_load+0x3fac>
   8:	01c000ef          	jal	ra,24 <main>

0000000c <dummy>:
   c:	00000013          	addi	zero,zero,0
  10:	00000013          	addi	zero,zero,0
  14:	00000013          	addi	zero,zero,0
  18:	00000013          	addi	zero,zero,0
  1c:	00000013          	addi	zero,zero,0
  20:	00000013          	addi	zero,zero,0

Disassembly of section .text:

00000024 <main>:
  24:	ff010113          	addi	sp,sp,-16
  28:	00100793          	addi	a5,zero,1
  2c:	00f12623          	sw	a5,12(sp)
  30:	00200793          	addi	a5,zero,2
  34:	00f12423          	sw	a5,8(sp)
  38:	00c12783          	lw	a5,12(sp)
  3c:	00812703          	lw	a4,8(sp)
  40:	00e787b3          	add	a5,a5,a4
  44:	00f12223          	sw	a5,4(sp)
  48:	01010113          	addi	sp,sp,16
  4c:	00008067          	jalr	zero,0(ra)

Disassembly of section .comment:

00000000 <.comment>:
   0:	3a434347          	fmsub.d	ft6,ft6,ft4,ft7,rmm
   4:	2820                	c.fld	fs0,80(s0)
   6:	29554e47          	fmsub.s	ft8,fa0,fs5,ft5,rmm
   a:	3920                	c.fld	fs0,112(a0)
   c:	322e                	c.fldsp	ft4,232(sp)
   e:	302e                	c.fldsp	ft0,232(sp)
	...

Disassembly of section .riscv.attributes:

00000000 <.riscv.attributes>:
   0:	2041                	c.jal	80 <_data_load+0x30>
   2:	0000                	c.unimp
   4:	7200                	c.flw	fs0,32(a2)
   6:	7369                	c.lui	t1,0xffffa
   8:	01007663          	bgeu	zero,a6,14 <dummy+0x8>
   c:	0016                	c.slli	zero,0x5
   e:	0000                	c.unimp
  10:	1004                	c.addi4spn	s1,sp,32
  12:	7205                	c.lui	tp,0xfffe1
  14:	3376                	c.fldsp	ft6,376(sp)
  16:	6932                	c.flwsp	fs2,12(sp)
  18:	7032                	c.flwsp	ft0,44(sp)
  1a:	5f30                	c.lw	a2,120(a4)
  1c:	326d                	c.jal	fffff9c6 <_bss_end+0xeffff9c6>
  1e:	3070                	c.fld	fa2,224(s0)
	...
