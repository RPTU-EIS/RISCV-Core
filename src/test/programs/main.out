13 01 10 11 EF 00 C0 01 13 00 00 00 13 00 00 00 
13 00 00 00 13 00 00 00 13 00 00 00 13 00 00 00 
13 01 01 FF 93 07 10 00 23 26 F1 00 93 07 20 00 
23 24 F1 00 83 27 C1 00 03 27 81 00 B3 87 E7 00 
23 22 F1 00 13 01 01 01 67 80 00 00 
